SeaPublicWebsite/Startup.cs

```diff
@@ -116,11 +116,13 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
             if (!webHostEnvironment.IsDevelopment())
             {
-                app.UseExceptionHandler("/Home/Error");
+                app.UseExceptionHandler();
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 // app.UseHsts();
             }
 
+            app.UseStatusCodePagesWithReExecute("/error/{0}");

```

Comment:
By doing so, on status code errors (i.e. 4xx, 5xx) we redirect to the /error route, which is the ErrorController. There we can handle them any way we want.